src/full_node/coin_store.py

```diff
@@ -105,6 +105,30 @@ async def get_coin_record(self, coin_name: bytes32) -> Optional[CoinRecord]:
             return CoinRecord(coin, row[1], row[2], row[3], row[4], row[8])
         return None
 
+    async def get_tx_coins_added_at_height(self, height: uint32) -> List[CoinRecord]:
```

Comment:
Should we pass in a list of heights to avoid repeated DB queries? This would help us support more wallet clients

Reply:
Not sure how to write that query